packages/jokul/package.json

```diff
@@ -455,6 +455,17 @@
                 "default": "./build/cjs/components/select/index.cjs"
             }
         },
+        "./styles/components/select-beta": "./styles/components/select-beta/_index.scss",
+        "./select-beta": {
+            "import": {
+                "types": "./build/es/components/select-beta/index.d.ts",
+                "default": "./build/es/components/select-beta/index.js"
+            },
+            "require": {
+                "types": "./build/cjs/components/select-beta/index.d.cts",
+                "default": "./build/cjs/components/select-beta/index.cjs"
+            }
+        },
```

Comment:
Hvis vi eksponerer komponenten gjennom den vanlige Select-pakken strenger vi vel heller ikke disse linjene (ser også ut som de peker på den gamle stien). Stilarket slipper vi kanskje ikke unna å ha en egen import for (men den stien må også oppdateres).